simra/database_test.go

```diff
@@ -42,9 +42,10 @@ func (db *mock) Get(key string) interface{} {
 }
 
 func TestMock(t *testing.T) {
-	db := OpenDB(&mock{}, filepath.Join("."))
-	if db == nil {
-		t.Error("failed to open database")
+	db, err := OpenDB(&mock{}, filepath.Join("."))
+	if err != nil {
+		t.Error("failed to open database. err: %s", err.Error())
```

Comment:
__[go vet]__

possible formatting directive in Error call


<!-- Comment by [SideCI](https://sideci.com) -->